code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm

```diff
@@ -15,23 +15,8 @@
 	layer = LARGE_MOB_LAYER //Looks weird with them slipping under mineral walls and cameras and shit otherwise
 
 /mob/living/simple_animal/hostile/megafauna/death(gibbed)
-	if(health > 0)
-		return
-	else
-		feedback_set_details("megafauna_kills","[initial(name)]")
-		..()
-
-/mob/living/simple_animal/hostile/megafauna/gib()
```

Comment:
Guess I was thinking of the staff or something then. Either way, you can still get more deathbolts on a regular basis from xenobio.

>I can put it in another PR

Or rather than convince you to not change something you hate, that's disabled on your server anyway, I can just fix the bug myself instead of negotiating for you to remove changes that were now twice undeclared in the op